docs/source/repair/repair-faster.rst

```diff
@@ -13,7 +13,7 @@ Speedup a running repair
 ========================
 
 If a repair is running on a cluster, you can change the intensity and parallelism level that should be applied while it is running.
-Run the following command to adjust the number of token ranges per Scylla repair job to the maximum supported (in parallel) by a repair master node.
+Run the following command to adjust the number of token ranges per Scylla repair job to the maximum supported (in parallel).
```

Comment:
You probably want to add a chapter that describes the relation between the "repair task" and "repair job" (each "task" runs up to `parallel` "jobs" in parallel).

Otherwise this and similar following places are very confusing.

Reply:
done